NEWS.md

```diff
@@ -33,6 +33,11 @@ from a BDS dataset, e.g. ADLB. (#943)
 
 ## Updates of Existing Functions
 
+- `impute_dtc` - Fixed imputation bug. A user setting `date_imputation = MID` and 
+`preserve = FALSE` would expect the date **2019---07** to be imputed to **2019-06-30**, 
+but the function was returning **2019-06-15**. Now returns it correctly. This bug fix 
```

Comment:
I would use `"2019---07"` instead of **2019---07** because `---` is rendered as a long dash (in the html documentation), which is confusing.